packages/modular-scripts/src/build.ts

```diff
@@ -0,0 +1,488 @@
+// While working on this file, be aware that builds
```

Comment:
I'm looking for feedback on whether the comments in this file are easy to understand, and if the code itself is grokkable.